doc/man/scons.xml

```diff
@@ -5937,20 +5926,18 @@ the entire directory tree
 will be removed.
 If the
 <parameter>must_exist</parameter>
-flag is set to a True value,
-then a Python error will be thrown
+flag is set to a true value,
+then a Python error will be raised
 if the specified entry does not exist;
-the default is False,
+the default is false,
 that is, the Action will silently do nothing
 if the entry does not exist.
 Examples:</para>
 
 <programlisting language="python">
 Execute(Delete('/tmp/buildroot'))
 
-env.Command('foo.out', 'foo.in',
-            [Delete('${TARGET.dir}'),
-             MyBuildAction])
+env.Command('foo.out', 'foo.in', [Delete('${TARGET.dir}'), MyBuildAction])
```

Comment:
I almost prefer the previous formatting.. this one breaks the action over two lines...